ledger-tool/src/main.rs

```diff
@@ -1872,6 +1873,10 @@ fn main() {
                 working_bank.print_accounts_stats();
             }
             println!("Ok");
+            let mut drop_time = Measure::start("drop_time");
+            drop(bank_forks);
+            drop_time.stop();
+            info!("drop(bank_forks): {}us", drop_time.as_us());
```

Comment:
This drop coincides with a long delay after ledger-tool verify is finished. Reporting this will help the next person verify that this portion of the validator code path is capturing time correctly. Without the explicit drop, the time is when bank_forks goes out of scope and is dropped.